<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TemplateWindow</class>
 <widget class="QMainWindow" name="TemplateWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1114</width>
    <height>865</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>樣板</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="topMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <item>
       <widget class="QTabWidget" name="tabWidget_charge_settings">
        <property name="currentIndex">
         <number>-1</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_update_nhi_payment"/>
   <addaction name="action_update_nhi_payment_old"/>
   <addaction name="separator"/>
   <addaction name="action_close"/>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1114</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>檔案</string>
    </property>
   </widget>
   <addaction name="menu"/>
  </widget>
  <action name="action_close">
   <property name="icon">
    <iconset>
     <normaloff>../icons/application-exit.svg</normaloff>../icons/application-exit.svg</iconset>
   </property>
   <property name="text">
    <string>關閉視窗</string>
   </property>
  </action>
  <action name="action_update_nhi_payment">
   <property name="icon">
    <iconset>
     <normaloff>../icons/view-refresh.svg</normaloff>../icons/view-refresh.svg</iconset>
   </property>
   <property name="text">
    <string>更新健保支付標準</string>
   </property>
   <property name="toolTip">
    <string>更新健保支付標準</string>
   </property>
  </action>
  <action name="action_update_nhi_payment_old">
   <property name="icon">
    <iconset>
     <normaloff>../icons/edit-undo.svg</normaloff>../icons/edit-undo.svg</iconset>
   </property>
   <property name="text">
    <string>還原舊健保支付標準</string>
   </property>
   <property name="toolTip">
    <string>還原舊健保支付標準</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
